code/hash/slice-vs-select-include.rb

```diff
@@ -0,0 +1,22 @@
+require 'benchmark/ips'
+
+HASH = {
+  title: "awesome",
+  description: "a description",
+  author: "styd",
+  published_at: Time.now
+}
+
+def fast
+  HASH.slice(:title, :author)
+end
+
+def slow
+  HASH.select {|k, _| [:title, :author].include? k }
```

Comment:
```
[:title, :author].each_with_object(Hash.new) { |k, h| h[k] = HASH[k] if h.has_key?(k) }
```
Based on [Active Support](https://github.com/rails/rails/blob/94b5cd3a20edadd6f6b8cf0bdf1a4d4919df86cb/activesupport/lib/active_support/core_ext/hash/slice.rb#L23-L25), which is a bit faster than this on my machine... but I don't think comparing native methods with their Ruby equivalent is really the point, of course that'll be faster.

Reply:
You are right. That's what I thought before making this PR.
But I PR anyway because:
1. To show other rubyist who doesn't already know that they have another alternative in newer ruby.
2. To show other rubyist who already know the `slice` method that it is not just a method that wraps the ruby select-includes.

Reply:
@styd Your `slow` implementation is basically simply bad code. You're creating an array on each key/par cycle. So it worth to replace `slow` method with something good ruby code would look like (pretty much what @bdewater provided).